### ISSUE TITLE
User is created as Normal User when should be SuperUser/Admin

### ISSUE THREAD

**Issue by joeywashburn** [ansible/awx#11886]
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

 - UI

##### SUMMARY
We have users that are in different domain that are not being added to Admin/Super User when logging in via LDAP.  They are in the correct group.  In addition they do get created and can get logged in, but are Normal User so the basics of LDAP are working.  It seems that maybe its order of operations, or group lookup is failing since the user is a different domain thn the group.  This is using Active Directory.  Setup to use LDAP in Settings

##### ENVIRONMENT
* AWX version: 20.0.0
* AWX install method: operator 
* AWX deployment target: kubernetes
* Operating System:
* Web Browser:

##### STEPS TO REPRODUCE
Create LDAP connection. 
LDAP Group Type ActiveDirectory (Also happens with MemberDNGroupeType)

User in question is in child domain europe.domain.com
LDAP Bind account is in usa.domain.com
User in question is a member of the group CN=awx_admins,OU=Groups,DC=usa,DC=domain,DC=com

LDAP User Search

[
  [
    "DC=usa,DC=domain,DC=com",
    "SCOPE_SUBTREE",
    "(sAMAccountName=%(user)s)"
  ],
  [
    "DC=europe,DC=domain,DC=com",
    "SCOPE_SUBTREE",
    "(sAMAccountName=%(user)s)"
  ]
]

LDAP Group Search

[
  "DC=usa,DC=domain,DC=com",
  "SCOPE_SUBTREE",
  "(objectClass=group)"
]

LDAP User Flags By Group
{
  "is_superuser": [
    "CN=awx_admins,OU=Groups,DC=usa,DC=domain,DC=com"
  ]
}
LDAP Organization Map
{
  "TESTING": {
    "remove_users": true,
    "users": [
      "CN=awx_users,OU=Groups,DC=usa,DC=domain,DC=com"
    ],
    "remove_admins": true,
    "admins": "CN=awx_admins,OU=Groups,DC=usa,DC=domain,DC=com"
  }
}

##### EXPECTED RESULTS

User is created with System Administrator permissions

##### ACTUAL RESULTS

User is created as Normal User

**Comment by mabashian** [comment]
Hey - any chance you could increase the logging threshold and report back the groups returned?

**Comment by joeywashburn** [comment]
Yes, I would be happy to.  Can you point me in the right direction of where
to bump it at

**Comment by joeywashburn** [comment]
I was able to bump the logging and have the user login.  Basically it
saying the user is not found in the group, yet I can look in AD Users and
Computers and see the user is actually in the group.  So not sure if its
LDAP problem, an AWX problem, or a django problem.

**Comment by rebeccahhh** [comment]
Hi @joeywashburn this sounds like a LDAP mapping issue. I think to help you out we'll need a little more info about the mapping that you have set up there. I understand that may not be something you can share but given our experience with other users checking that relationship is the best place to start.

**Comment by joeywashburn** [comment]
With this new information I was able to find
https://github.com/ansible/awx/issues/3757

Looks like this has been reported before
